orderbook/src/orderbook.rs

```diff
@@ -51,6 +53,25 @@ impl OrderBook {
             Err(RemoveOrderError::DoesNotExist)
         }
     }
+
+    // Clean up operations like removing expired orders.
+    pub async fn run_maintenance_forever(&self) -> ! {
+        loop {
+            tracing::debug!("running order book maintenance");
+            self.remove_expired_orders(SystemTime::now()).await;
+            tokio::time::delay_for(MAINTENANCE_INTERVAL).await;
+        }
+    }
+
+    async fn remove_expired_orders(&self, now: SystemTime) {
+        // TODO: use the timestamp from the most recent block instead?
```

Comment:
This is a discussion point. If we prefer using the block then I still want to merge this PR as is and do that change in another PR where I would also add web3 as a dependency.l

Reply:
I think the local time is perfectly fine.